zello-sdk-sample-contacts/src/main/AndroidManifest.xml

```diff
@@ -17,4 +19,14 @@
             </intent-filter>
         </activity>
     </application>
+
+    <!-- Declare intents and packages that the app would like to have access to on Android R+ -->
+    <queries>
+
+        <package android:name="com.loudtalks" />
+
+        <package android:name="net.loudtalks" />
+
+        <package android:name="com.pttsdk" />
+    </queries>
```

Comment:
I think you should be able to include this in the library manifest rather than it needing to be in the samples and every app that uses the library, correct?

Reply:
Not the way it stands now. We ship a jar, not an aar, and jars only include class files...

Reply:
Maybe we should switch AAR. JAR was an apparent simple choice, also AARs didn't exist at the time.